.github/workflows/cd.yml

```diff
@@ -2,22 +2,31 @@ name: CD
 
 on:
   workflow_dispatch:
+    inputs:
+      network:
+        default: 'Mainnet'
+      checkpoint_sync:
+        default: true
   push:
-    branches:
-      - main
+    # branches:
+    #   - main
 
 env:
+  CARGO_INCREMENTAL: '1'
+  NETWORK: Mainnet
```

Comment:
I think this is the default network that is used if the network input isn't set.
But there's also a default in the input part of the workflow.

Can we remove it if it is redundant, to avoid confusion?
(Other workflow configs don't have this env var.)

Reply:
We have it twice as this is a single yml which now has the manual `workflow_dispatch` and the automated workflow from `push` to main.

If there won't ever be a need to change this from Mainnet to Testnet in the manual deployment, we can surely leave just the env variable to handle it here: https://github.com/ZcashFoundation/zebra/blob/5989bb992537fc6b63f8cad7101ce956266e5bd3/.github/workflows/cd.yml#L77

Reply:
That makes sense. We might want to manually deploy to testnet, so let's leave it as it is.

Can you document what each duplicate setting does, so we don't accidentally remove them?